src/pevm.rs

```diff
@@ -328,6 +262,54 @@ pub fn execute_revm_sequential<S: Storage>(
     Ok(results)
 }
 
+fn canonize_account(
+    results: &mut [PevmTxExecutionResult],
+    address: Address,
```

Comment:
For performance, it's better to pass in a list of addresses (`lazy_addresses = [beneficiary_address] <> lazy_tos <> lazy_senders <> ...`) here to call this function only once. That way, the compiler can confidently inline this function and remove indirections.

Reply:
This function is very small after refactoring so I wouldn't mind inlining it, to be frank.